bin/check_mypy_annotations.py

```diff
@@ -109,6 +109,11 @@ def process_source(lines, added_lines, line_to_func_map):
         if source_utils.is_line_annotated(first_line):
             continue
 
+        end_line_of_def = lines[first_line_num - 1]
```

Comment:
Would it be possible to move this into `source_utils.is_line_annotated`? That way all of the tools will get support for this ^_^